### PR TITLE
Barrier Focus States

### PR DESCRIPTION
**Pull request by lc512k**
This PR fixes focus states in the trial barrier. 

- Instead of focusing on individual elements we now focus on the entire offer column
- Focus state is now more visible
- Entire column is clickable
- Redundant links hidden from screen readers
- More context added to headings
- Improvement for "the rest of us" entire column is now clickable, so clicking slightly outside of the image still takes you to the sign up form

CSS changes in https://github.com/Financial-Times/n-barriers/pull/49

test

cc @keirog @i-like-robots @glynn

@tgabe tracking ids were moved to the full block, which is the only clickable element now, so stats shouldn't be affected. All changes should be mirrored in FastFT barrier.

### CODE REVIEW

File: views/partials/barriers/trial-grid.html [hunk 1]
```diff
@@ -41,7 +42,7 @@ <h2 class="u-visually-hidden">Subscribe to FT.com</h2>
 					{{#each offers}}
 					<div class="trial-barrier-grid__option trial-barrier-grid__option--option{{@index}} {{#if this.discount}}trial-barrier-grid__option-discount{{/if}}">
 							<div class="trial-barrier-grid__option-inner">
-								<h3>
+								<h3 aria-hidden="true">
```
Comment (keirog) [inline views/partials/barriers/trial-grid.html]: I’m curious about this bit—how come we’re hiding the heading?

Comment (keirog) [inline views/partials/barriers/trial-grid.html]: Ah I see what the test is now

**Comment by lc512k** [comment]
oy, sorry, this was the least verbose PR ever. Still needs more work. I'm hoping to knock it off the list today. 

WIll add a description and cc you for review in a bit @keirog

**Comment by keirog** [comment]
All looks good but I can’t see where your `:focus` selector is… is that coming next?

**Comment by lc512k** [comment]
@keirog https://github.com/Financial-Times/n-barriers/pull/49

I think the css styles were originally put in a separate repo (`n-barriers`) because they're shared with the barriers we show in fastFT. All of this needs rethinking, it's fairly old, and barriers will be redesigned at some point this quarter.

**Comment by keirog** [comment]
Ah I see, ok ta!

**Pull request merged**